src/dialogs/config-flow/step-flow-pick-handler.ts

```diff
@@ -79,6 +80,22 @@ class StepFlowPickHandler extends LitElement {
             `
         )}
       </div>
+      <p style=${styleMap({ textAlign: `center` })}>
+        <span style=${styleMap({ display: `block` })}>
```

Comment:
A `span` with `display: block`, that's a `p` right?

Reply:
Yeah you are right, I probably made it more difficult than it needs to be. However, if I put everything into one `p` it will not automatically handle the line break after the first sentence. Which does not look very pretty:
![grafik](https://user-images.githubusercontent.com/46536646/66715254-44305500-edc1-11e9-8e3b-e6f9767fd2ae.png)
Any suggestions on how to deal with this problem?

Reply:
Use a line break `<br>`

Reply:
Use a line break `<br>`

Reply:
Thought that using them was not optimal but sure I can do that.
Hopefully a9ff740f0c361f3aa376c358e6e2aaa95304f7d8 addresses all your recommended changes :)